presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapConfig.java

```diff
@@ -49,6 +51,19 @@ public LdapConfig setLdapUrl(String url)
         return this;
     }
 
+    public File getTrustCertificate()
+    {
+        return trustCertificate;
+    }
+
+    @Config("ldap.ssl-trust-certificate")
```

Comment:
forgot to update this comment; I know you don't accept password because you support only PEM in this code.
It is probably fine for now, but i am concerned it may be not sufficient in all deployments, just like ended up having support for password in few other places.

Reply:
I'd rather phase out support for JKS. If we need to support it later, we'll deprecate this and switch to the other name style. Regarding `tls` vs `ssl`, I chose `ssl` since that matches the majority of existing configs and language used in the documentation.

Reply:
> Regarding tls vs ssl, I chose ssl since that matches the majority of existing configs and language used in the documentation.

uh, i didn't do statistics, sorry. But I think we should go SSL->TLS direction in wording (except for code places where things like SSLContext are in stone).